lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -729,6 +729,13 @@ void ResourceMap::loadDefaultConfiguration()
 
   // Contour parameters //
   addAsUnsignedInteger("Contour-DefaultLevelsNumber", 10);
+  addAsBool("Contour-IsFilled", false);
+  addAsBool("Contour-DrawLabels", true);
+  addAsString("Contour-Norm", "linear");
+  addAsString("Contour-ColorMap", "viridis");
```

Comment:
I remembered Regis not agreeing to the cmap change,maybe the default should be kept to hsv ?
https://matplotlib.org/stable/gallery/color/colormap_reference.html

Reply:
Ok, I will switch the default from "viridis" to "hsv" in a later commit, although I expect the results to be hideous (at least for filled contour plots).

Reply:
I have tried it. Here is a comparison on the results on the `plot_graphs_loglikelihood_contour.py` example:

# Empty contour plot

Here, because of the "linear" norm, the cycling problem of HSV makes large values indistinguishable from small values.

## HSV

![image](https://github.com/openturns/openturns/assets/31989332/b840fb60-c69f-497d-99c9-f64a73e22fc2)

## Viridis

![image](https://github.com/openturns/openturns/assets/31989332/6ecfeba3-430d-40e2-9b0c-286c14f06f22)


# Filled contour

The HSV cycling problem seems to be less acute with filled contours, except that the artifact on the bottom left-hand corner is hidden, whereas it appears clearly with Viridis.

## HSV

![image](https://github.com/openturns/openturns/assets/31989332/4619b44e-5ba3-4e42-8ef7-663858ecced4)

## Viridis

![image](https://github.com/openturns/openturns/assets/31989332/2c43a01b-c2a3-4d4c-9d79-693123eb4bc9)

# Conclusion

I agree to leave HSV as the default, but it it needs to be manually changed in some examples.